IdentitySdkCore/IdentitySdkCore/Classes/ReachFive.swift

```diff
@@ -94,4 +94,102 @@ public class ReachFive: NSObject {
             .authWithCode(authCodeRequest: authCodeRequest)
             .flatMap({ AuthToken.fromOpenIdTokenResponseFuture($0) })
     }
+    
+    public func signupWithWebAuthn(profile: ProfileWebAuthnSignupRequest,origin: String,friendlyName: String?,viewController: UIViewController,scopes: [String]?, completion: @escaping ((Future<AuthToken, ReachFiveError>) -> Any)) {
```

Comment:
The `completion` parameter supplied by the library user is propagated down the call stack. Can't we simply use `Future`s all along and return a `Future<AuthToken, ReachFiveError>` to the caller so they can handle it however they want ? The code showing the error to the user below could then be deleted since it would be the responsibility of the caller.

Reply:
it's not a parameter it's a type of return, instead of putting a Future <AuthToken, ReachFiveError> return at the end of the method, we use the completion to return the value as soon as it is ready (because we waits for browser excustion) and it's always the same Future <AuthToken, ReachFiveError> type for integrators